frameworks/nexus/package.json

```diff
@@ -10,15 +10,16 @@
     "dev": "ts-node-dev --no-notify --respawn --transpileOnly src/server"
   },
   "dependencies": {
-    "@prisma/client": "2.0.0-alpha.1311",
+    "@nexus/schema": "^0.14.0",
+    "@prisma/client": "^2.0.0-beta.4",
     "graphql": "14.6.0",
     "graphql-yoga": "1.18.3",
-    "nexus": "0.12.0",
-    "nexus-prisma": "0.11.1"
+    "nexus-prisma": "^0.14.0"
   },
   "devDependencies": {
-    "@prisma/cli": "2.0.0-alpha.1311",
+    "@prisma/cli": "2.0.0-beta.4",
```

Comment:
This won't work as the e2e test main functionality is that they update this version (and client above) every time we release a new version of that. So renovate would just create a new PR again. 

Or is that what we want as this combination here should work, and if the update does not we know that nexus schema is not ready yet?

Reply:
I'm not familiar with alphas. I know beta 4 is what the current prisma plugin stable versions are suing. Sounds like this system is going by its own strategy though so no interest in going against it.

Reply:
Actually pardon me, technically `nexus-prisma` needs beta 4, anything else is outside what it obliges itself to work against.

Reply:
None the less I've reverted my changes.